Cargo.toml

```diff
@@ -163,27 +163,27 @@ ruff_python_ast = { git = "https://github.com/astral-sh/ruff.git", tag = "0.11.0
 ruff_text_size = { git = "https://github.com/astral-sh/ruff.git", tag = "0.11.0" }
 ruff_source_file = { git = "https://github.com/astral-sh/ruff.git", tag = "0.11.0" }
 
-ahash = "0.8.11"
+ahash = "0.8"
 ascii = "1.1"
-bitflags = "2.9.1"
+bitflags = "2.9"
 bstr = "1"
 cfg-if = "1.0"
-chrono = "0.4.39"
+chrono = "0.4"
```

Comment:
We are using chrono features of later versions. e.g. 0.4.10 is not compatible. This will cause dependency resolver success but build fails.

Reply:
fixed

Reply:
How about other libraries? To remove patch version, we have to ensure it is buildable with x.y.0